src/Components/AppShell/AppShell.js

```diff
@@ -119,7 +119,7 @@ class AppShell extends Component {
             {this.props.children}
           </Scrollbars>
         </div>
-        <OnlyIf test={this.props.core.systemError != null}>
+        <OnlyIf test={this.props.core.systemError != null && this.props.core.showModal}>
```

Comment:
This is definitely future proofing here. Even if `showModal` currently is set to true only when there is a `systemError`, if for some reason `systemError` is not set in the future, then `BugModal` will have empty data. This is just being extra cautious, because why not?

I chose the name for `showModal` because `showBugModal` is awfully specific. It seems silly to have a booleans for `showBugModal`, `showUpdateModal`, `showNoticeModal`, `showXXXXModal`. Again just trying to be generic.

Reply:
I will just go ahead and use showBugModal. We'll generalize it later if we need to.